### ISSUE TITLE
nox does not work when installed via pipx (Windows, Python 3.7.3)

### ISSUE THREAD

**Issue by pfmoore** [pipxproject/pipx#123]
There appears to be a problem with nox when run from via pipx:

```
>nox
nox > Running session test
nox > Creating virtualenv using python.exe in C:\Work\Scratch\n\.nox\test
nox > Command c:\users\pfmoore\.local\pipx\venvs\nox\scripts\python.exe -m virtualenv C:\Work\Scratch\n\.nox\test failed with exit code 100:
No pyvenv.cfg file
Using base prefix 'C:\\Users\\pfmoore\\AppData\\Local\\Programs\\Python\\Python37'
New python executable in C:\Work\Scratch\n\.nox\test\Scripts\python.exe
ERROR: The executable C:\Work\Scratch\n\.nox\test\Scripts\python.exe is not functioning
ERROR: It thinks sys.prefix is 'c:\\work\\scratch\\n' (should be 'c:\\work\\scratch\\n\\.nox\\test')
ERROR: virtualenv is not compatible with this system or executable
Note: some Windows users have reported this error when they installed Python for "Only this user" or have multiple versions of Python installed. Copying the appropriate PythonXX.dll to the virtualenv Scripts/ directory may fix this problem.
```

This seems to be related to mixing venv and virtualenv - see https://github.com/pypa/virtualenv/issues/1339. If so, it will probably affect other tools that use virtualenv, like tox, pew and pipenv.

**Comment by cs01** [comment]
Thanks for sharing. pipx exclusively uses venv. My limited testing with pipenv seemed to work fine, but good to know there might be some conflicts or issues.

**Comment by pfmoore** [comment]
No problem - I'm not sure where a fix would lie here, it seems to be a relatively new (and in this case Windows-specific) variation of a long-standing inconsistency between how virtualenv and venv work, so it may take some time to identify a suitable fix. In the meantime, knowing that there are potential issues with tools that use virtualenv might be as much as we can do.

**Comment by cs01** [comment]
Sounds good, I will add a note to the README saying that, so people don't have to search around the issues.

**Comment by pfmoore** [comment]
This is fixed now with the release of virtualenv 16.5.0. People with existing venvs created with pipx will need to ensure they get the new version of virtualenv into those venvs (and we can't do anything about tools that pin to older versions of virtualenv, of course) but there's no further need to track this here, IMO.

To be precise, if a user did (for example) `pipx install nox` and got the broken version of virtualenv, they will now need to run `pipx runpip nox install --upgrade virtualenv` to get the new version.
